app/assets/stylesheets/govuk-frontend/extensions.scss

```
@@ -35,3 +35,16 @@
 .govuk-link--destructive {
   @include govuk-link-style-destructive-no-visited-state;
 }
+$govuk-grid-widths: (
+  one-eigth: 12.5%,
+  one-sixth: 16.6667%,
```

Should round this down otherwise `6 * one-sixth > 1`
```suggestion
  one-sixth: 16.6666%,
```